CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Released: TBD
 
 ### Bug Fixes
 
+- [#347](https://github.com/peggyjs/peggy/issues/347) Disallow '$' as an initial
+  character in identifiers.  This is not a breaking change because no grammar
+  could have successfully used these in the past.  From @hildjj.
+
```

Comment:
Actually, breaking, because you still can _start_ from such rule, but I think we can ignore that